sign.proj

```diff
@@ -39,6 +39,15 @@
     </ItemGroup>
 
     <ItemGroup>
+      <FilesToSign Include="$(OutDir)$(ConfigurationGroup)/**/Microsoft.DotNet.PlatformAbstractions.dll">
```

Comment:
The way this used to work is that we'd put the build result in the `$(IntermediateOutputPath)forPackaging` folder.  See below:

```
     <ItemGroup>
       <FilesToSign Include="$(IntermediateOutputPath)forPackaging/**/*.dll">
         <Authenticode>$(CertificateId)</Authenticode>
       </FilesToSign>
     </ItemGroup>
```

and here:  https://github.com/dotnet/core-setup/blob/v2.0.0-preview1/build_projects/dotnet-host-build/PackageTargets.cs#L216

I think we should set the BaseOutputPath to that folder and then we wouldn't have to change signing.proj at all.